README.md

```diff
@@ -68,6 +68,7 @@ deployments/                     contains files used for deployment technologies
 docs/                            contains documentation about the project
 examples/                        contains additional `Dockerfile` examples that extend the base 
                                  configuration
```

Comment:
Are there any notes we should make about editor integration or docs generation? Do we need to add a note about the nodejs requirement?